app/src/main/java/com/myperishableplanner/v21001/service/ItemService.kt

```diff
@@ -17,9 +17,8 @@ class ItemService : IItemService {
     override suspend fun fetchItems() : List <Item>? {
         return withContext(Dispatchers.IO) {
             val service = RetrofitClientInstance.retrofitInstance?.create(IItemDAO::class.java)
-            val items = async { service?.getAllItems()}
-            var result = items.await()?.awaitResponse()?.body()
-            return@withContext result
```

Comment:
The return statement can be condensed down to one line.

Reply:
The downside of putting too much in one line... if you get an error on that line, it makes it harder to determine the root cause.